File: engine/model/src/main/scala/tools/packet/guild/GuildCapacityChange.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GuildCapacityChange(private var _guildId: Int, private var _capacity: Int) extends PacketInput {
  def guildId: Int = _guildId

  def capacity: Int = _capacity

  override def opcode(): SendOpcode = SendOpcode.GUILD_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/guild/GuildCapacityChange.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GuildCapacityChange(Integer guildId, Integer capacity) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_OPERATION;
   }
}